stack/keycloak/11_johndoe_webapp.yml

```diff
@@ -0,0 +1,13 @@
+id: johndoe_assign_webapp_role
+author: Simon Zambrovski
+realm: ${SSO_REALM}
+changes:
+  - addRole:
+      clientId: ${SSO_ENGINE_CLIENT_ID}
+      clientRole: true
+      name: digiwf-webapp-user
```

Comment:
Should we give him the camunda-admin role?

Reply:
`camunda-admin` is not specific. we can call it anyhow and need to configure the authorizations for it.

Reply:
Ok, should we give him the "admin" role. so we have local no problems when working with authorization.enabled in the engine?